src/api/mod.rs

```diff
@@ -1,5 +1,6 @@
 //! `Web3` implementation
 
+mod accounts;
```

Comment:
Please re-export `accounts::Accounts` otherwise the docs are not visible.

Reply:
Oops!